packages/ui/src/ui/store/actions/chaos_cell_bundles/index.ts

```diff
@@ -43,13 +44,15 @@ export function fetchChaosBundles(): ChaosBundlesThunkAction {
                 parameters: {
                     path: '//sys/chaos_cells',
                     attributes: ['health', 'id', 'peers', 'status', 'tablet_cell_bundle'],
+                    max_size: USE_MAX_SIZE,
```

Comment:
...USE_MAX_SIZE
it looks like we need to describe max_size field for `list` in `BatchSubRequest`. And it is better to add explicit type for `requests`:
```ts
const requests: Array<BatchSubRequests> = [
    ...
]
```